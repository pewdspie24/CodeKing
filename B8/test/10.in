20
49
756773652 276044581 203778359 37168823 918411368 226442525 722457275 816018495 874551611 869587068 784291297 30957921 495381055 456417588 548060601 104015948 249699710 822601738 402342498 725379995 158431258 209955661 99991488 542244086 602840313 816308228 720862030 938265679 900645278 810385112 905015964 293736740 733918102 677909036 411070797 52031100 347876501 819312131 303717534 575823868 527148887 294224403 212939927 674956417 773074211 75290413 987601947 338136382 2121752 
49
296412867 517689957 629705505 576564145 320358664 728309700 126126215 1314092 245694295 793555826 313288459 773109951 520907656 521033443 22441673 492499869 704789121 71065570 419724975 413293828 439327586 776186748 815843189 572175597 997766151 707166959 819332947 681772215 443606259 835078548 3829615 471430942 7072346 812801737 456641726 106604273 766545366 936561532 536380298 321910474 200631719 196334078 548488991 166310596 788806613 47047364 189952834 519220022 646627291 
44
541546752 180176015 577732347 472513260 79274232 978424809 756141417 908133746 204813754 247525245 969564214 696804308 863210619 35597863 159043847 348595593 147362543 674162783 281396128 224593274 136944256 30753624 649454660 146819089 643795123 507864892 633872325 233119540 610669589 941318908 571886253 809281503 324091765 900942118 259767578 245896437 138825106 232910031 50793639 369810282 28661792 457065317 629154654 53614825 
42
475425019 86980877 458100527 225687705 167908829 301685157 529221752 307753694 143492803 412634307 936736138 839676685 456952233 598937295 367856552 841987542 525496069 560666056 464888361 686515507 799470881 694635809 711404891 402961988 49434097 69299667 504508614 46075527 210110119 135005735 418113550 900240812 734265332 146038533 457892376 592246695 380225882 447814667 11786449 307269646 482576973 311559981 
45
498157826 511511994 45620560 194593458 580856581 679390973 868946309 927154744 470766547 410141603 468621316 18283915 275060662 869276933 951624385 303599001 917578218 385546959 483459487 892427155 461962332 683509796 531972372 397247567 342507513 462392371 440058029 931700312 854479030 845096136 718387317 566976787 534118017 707999665 64292471 273690822 792110454 298857491 252361216 771692354 256263127 544130389 96077892 655472208 223619678 
48
961581063 289885699 187841693 86822746 901628530 916231641 940157058 844573893 799074750 964124130 737895266 71011049 647318527 906260488 442046362 780673079 467906865 116352554 936627825 287311666 969032156 106817447 118441182 232650038 536160832 182059785 217600413 399679378 521584349 108765345 695743366 242435051 265077646 806649021 51921536 12499942 25814070 683721630 303142231 581011408 111521304 967968073 405235450 722644894 287291765 549546264 124903052 10486613 
38
872083050 34920382 333039213 266432590 811575136 302513915 267550687 757234909 162126841 579092512 273869899 581788339 176986167 12293142 468003303 624156620 959377298 999568689 758623757 851456098 851306155 20038164 177805078 646438153 17089609 350495828 755317731 261735003 661577483 817106836 18092751 879901351 598522236 13113470 218974428 532559018 707350040 444991419 
24
713110147 245944906 705090933 390544659 541886721 399290600 420619097 480301951 474026420 684532407 76168349 348525455 474055630 959475191 451482518 550239315 864604540 474877724 405555797 11370899 479414078 445835109 886655584 336072851 
44
571953233 569955480 372193329 37574840 745891935 462265059 399601906 880428598 949045761 182507927 839125453 402835824 39327503 41368893 254975245 599909724 664401820 677038912 929956492 598330112 874335528 59393013 715473614 597209059 62472219 805496918 744525026 627450391 157013652 435650352 287349699 132558408 300536194 592139346 657951782 314659215 113674690 831847855 137581934 577181388 569617747 892713207 835436948 465998371 
32
638731639 620631390 805770281 211506921 722517846 740489344 78710928 832174871 490242156 68454536 438030004 293076969 890018849 112297417 12970587 826319268 119596873 292258876 748552001 845792805 981131259 429380751 836386581 141868242 766375829 463200273 838626687 507611165 440399628 636802553 27426285 924756093 
48
181998717 19841450 56173322 539877050 539213990 529060889 69481312 235708752 406136169 877575909 715737972 49945335 204885805 45189887 505566291 230690568 379794844 355227375 303367011 809217460 540501432 644299946 687163509 839400568 203934864 131118562 98468586 223619253 165801279 170881176 559604201 423995837 141583381 809405027 773975573 314351271 146817300 115101753 361436260 152243842 252167208 226507524 485171931 93579387 976576862 77344682 325484314 545080208 
21
925233327 38406977 93970768 188610913 763248170 963540898 291999949 298466466 96146460 985837019 728848681 809560821 683120713 702368316 308512259 458382251 461665287 136096096 530670579 773541100 202147617 
49
389495715 405851610 893349191 895655656 379744205 200897906 328529836 379627566 806653037 15231787 297918963 152394695 616360099 287507581 731602304 229651027 22975748 360605266 176784842 875604103 269442991 807709240 796479885 21871584 787268555 816263511 335413932 870961182 648407124 436909702 986043165 71834141 787208187 427313138 601567587 519348027 911140634 596382114 634511373 822379365 921890794 396856520 568955757 849923519 267523687 862300023 279672921 23527635 886870839 
24
320091547 676853646 907118901 122363286 206606886 5793246 159794974 852549818 812662807 799251687 268316929 439577526 45644370 857179200 485647462 824078019 55847581 609148528 810556585 86998423 444427306 728460441 528117642 183199845 
36
32274977 976829953 200208857 865992960 350567122 826510647 230841808 289917808 681315214 828732994 154787784 93465511 743153651 55568321 995339300 369685290 291962387 90399890 990554283 408525389 976172079 71053672 883121432 399295242 159956321 255186512 427854006 608090320 208541330 385402888 273530257 798182827 505980161 69646753 154368682 133311935 
48
436024661 891259040 375314813 30522908 231958794 213636584 739894342 672755071 246411975 788823170 108846172 201075225 205055891 257930486 65913668 475587479 963339103 515755931 922298710 430104268 919101240 721935394 97101958 123796010 205178962 492653127 675762239 204029156 41532800 96398851 718357044 960959972 573711852 337307443 497887123 421707968 330297618 293990853 638262285 841255625 9347939 17755470 58167486 72824351 711422213 974454806 847765770 766681045 
34
5185310 442072612 808113064 657045218 584870174 296594249 917800910 231328987 831964572 508661838 60460877 800649789 189755379 692970897 682681509 612916221 276240040 866323317 229560884 727783129 370249986 874529127 25849611 854765717 918050992 39195407 772951180 451607412 146212014 800677446 436032940 209410132 392953847 681155475 
26
190252193 705333005 837713004 972254502 841683917 71322020 523502349 369749060 244607369 172914160 524467105 443718978 383406554 445349119 685693000 577094903 552537503 3182914 584462671 592482004 808919828 352086398 551049447 621992964 138473256 563324533 
24
90799878 105689568 176308670 809413922 500850367 687618493 209604540 639758661 318742906 329150570 760396762 879366738 536004533 610929753 134701300 160686207 608742153 498986185 854221441 1229553 113146958 912706800 306239155 493111259 
47
660220413 312339127 67824013 335245481 736674696 125147948 379386077 615100973 50485795 5256979 500226896 54584397 578375640 996945505 681449991 436920965 337235870 240929502 782234137 385159156 634061129 46103240 807516988 345766235 248375320 264675969 305271154 246611385 125948323 474669076 476481714 276085457 882100994 247511117 540187302 195959499 56878492 336058623 389553259 695631148 913748602 598183826 441414915 937330897 644337875 838353492 258146262 
