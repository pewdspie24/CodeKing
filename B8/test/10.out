Bi
Bi
Lema
Bi
Bi
Bi
Lema
Lema
Lema
Lema
Bi
Bi
Bi
Lema
Bi
Bi
Bi
Bi
Bi
Bi
